src/app/shared/notifications/notification/notification.component.scss

```diff
@@ -30,3 +31,9 @@
 .alert-warning .notification-progress-loader span {
   background: var(--ds-notification-bg-warning);
 }
+.alert-success{
+	color: darken($success, 22%);
+}
+.alert-danger {
+  color: darken($danger, 22%);
+}
```

Comment:
If we use sass variables in component styles, they won't be affected by theme changes. That's why everything has been replaced with css variables: they do update on the fly. 

So I'd either turn these color into `ds-` variants and move them to custom_variables.scss, or, since these alert classes are global classes that aren't unique to the notification component anyway, it might make more sense to put these in global-styles.scss, where you _can_ use sass variables (you can anywhere inside the styles directory)